data/remnant/remnant 2 cognizance.txt

```diff
@@ -666,11 +664,11 @@ mission "Remnant: Cognizance 19"
 	landing
 	name "Deliver navigation data"
 	description "Deliver navigational data on the new systems to <destination>."
-	source
-		government "Remnant"
+	source "Aventine"
```

Comment:
```suggestion
	source
	 	 government "Remnant"
	 	 neighbor planet "Aventine" "Caelian"
```

This mission was intentionally setup (and written) with the intention that it could trigger on any Remnant location, including stations, and any other permanent bases that are revealed in future content. A better source filter, for now, would be a Remnant planet that is neighbor to Aventine or Caelian. (with both listed, that should enable everywhere in the core trio of systems.

I'm not 100% that I did the neighbor correctly, but hopefully I did. The wiki says multiple planets can be listed in sequence.

Reply:
If that's what you want, it should probably just be:
```
government "Remnant"
near <any of those three systems> 1
```
But, given that this mission can offer as soon as Cognizance 18 is complete, which occurs on Aventine, there's not much reason to have this mission offer anywhere else.

Reply:
OK, updated the suggestion above. Thanks!

Reply:
There's still the fact that this mission will *never* offer anywhere but Aventine anyway, unless you specifically exclude Aventine.

Reply:
So the fact that it could, theoretically, if the player managed to get there somehow, is in this case more of a safety feature in case someone manages to break something or some corner case we aren't aware of results in it not triggering there this one time.

Unless this is actually a problem, I don't see a reason why allowing it to trigger elsewhere (as a safety feature if nothing else) should be an issue.

Reply:
What it does is unnecessarily obfuscate what is actually going on. We shouldn't make stuff needlessly complicated as a safety feature in case something breaks, especially when you cannot even describe the mechanism that would have to break. Even if the breakage you suggest occurred, a person would still be able to get this mission upon returning to Aventine anyway.

Reply:
All it takes is one mission that for some reason takes priority here to block this from happening. 

As far as that goes, I've had this not trigger the first time. No idea why (hasn't happened recently, so I can't provide circumstances or saves), but in my experience the logic for missions firing (especially since the game doesn't let more than one mission offer in the same place at a time) isn't iron-clad.

That being said "any Remnant planet within 2 jumps of Arculus" is hardly obfuscated or hard to understand.

Reply:
> All it takes is one mission that for some reason takes priority here to block this from happening.

`priority` does not affect landing missions.

The letter thing is well-defined behaviour that people are taking advantage of. Not a vague "if something breaks."

But, actually, I have thought of a reasonable way this could be prevented from offering, though I don't think this arrangement is possible in the vanilla game right now, so sure. I do have other issues with this mission and the following ones which I've outlined in a comment near the top of this, but I don't want to deal with changing that stuff right now.

Reply:
alright.